corehq/form_processor/change_publishers.py

```diff
@@ -34,3 +40,15 @@ def change_meta_from_sql_case(case):
         domain=case.domain,
         is_deletion=False,
     )
+
+
+def publish_case_deleted(domain, case_id):
+    producer.send_change(topics.CASE_SQL, ChangeMeta(
+        document_id=case_id,
+        data_source_type=data_sources.CASE_SQL,
+        data_source_name='case-sql',  # todo: this isn't really needed.
+        document_type='CommCareCaseSql',  # todo: should this be the same as the couch models?
+        document_subtype=None,  # todo: does this need a value?
```

Comment:
I think it's fine. Currently `subtype` is barely used (just in custom reports) and I think we can just make deletion logic not dependent on it